policyengine_us/variables/gov/states/ca/cdss/state_supplement/payment_standard/ca_state_supplement_aged_disabled_amount.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class ca_state_supplement_aged_disabled_amount(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "California SSI state supplement aged disabled amount"
+    unit = USD
+    definition_period = MONTH
+    defined_for = StateCode.CA
```

Comment:
This will eval to true if >0, improving performance
```suggestion
    defined_for = "ca_state_supplement_aged_disabled_count"
```